src/makefile

```diff
@@ -84,13 +87,13 @@ ifeq ($(NODENAME), cheyenne)
     ifeq ($(LMOD_FAMILY_COMPILER),gnu)
         FC=gfortran
 		COMPILER=gnu
-        NCDF_PATH=/glade/u/apps/ch/opt/netcdf/4.4.1.1/gnu/6.2.0
+        # NCDF_PATH=/glade/u/apps/ch/opt/netcdf/4.4.1.1/gnu/6.2.0
         LAPACK_PATH=/glade/u/apps/ch/opt/netlib/3.7.0/gnu/4.8.5
     endif
 	ifeq ($(LMOD_FAMILY_COMPILER),intel)
 	    FC=ifort
 		COMPILER=intel
-	    NCDF_PATH=/glade/u/apps/ch/opt/netcdf/4.4.1.1/intel/17.0.1
+	    # NCDF_PATH=/glade/u/apps/ch/opt/netcdf/4.4.1.1/intel/17.0.1
```

Comment:
let's just remove these if they are no longer relevant.

Reply:
Agreed, that was just laziness on my part until I had a chance to test it.